web/gds-admin-ui/src/pages/app/dashboard/Status.js

```diff
@@ -1,16 +1,24 @@
-// @flow
 import React from 'react';
 import { Doughnut } from 'react-chartjs-2';
 import { Card, Row, Col } from 'react-bootstrap';
+import { convertCountsToPercentages } from '../../../utils';
 
-const Status = (): React$Element<any> => {
+const Status = ({ statuses }) => {
     const colors = ['#0acf97', '#727cf5', '#fa5c7c'];
 
+    const getDonutChartData = () => {
+        if (statuses && typeof statuses === "object") {
+            return Object.values(convertCountsToPercentages(statuses))
+        }
+
+        return []
+    }
+
     const donutChartData = {
         labels: ['Completed', 'In-progress', 'Behind'],
```

Comment:
These labels also need to be updated to the statuses:
![Screen Shot 2021-09-24 at 17 23 21](https://user-images.githubusercontent.com/745966/134745624-25f77a9a-6092-43d0-8156-7f3308f67757.png)